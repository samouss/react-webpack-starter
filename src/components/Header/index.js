import React from 'react';
import PropTypes from 'prop-types';
import './index.css';

const Header = ({ title }) => (
  <h1 styleName="container">
    React Starter, {title}
  </h1>
);

Header.propTypes = {
  title: PropTypes.string.isRequired,
};

export default Header;
