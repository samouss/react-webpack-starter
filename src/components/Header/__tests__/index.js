import React from 'react';
import { shallow } from 'enzyme';
import Header from '../index';

describe('<Header />', () => {
  it('should render the component', () => {
    const props = {
      title: 'Test app',
    };

    const component = shallow(
      <Header
        {...props}
      />,
    );

    expect(component).toMatchSnapshot();
  });
});
