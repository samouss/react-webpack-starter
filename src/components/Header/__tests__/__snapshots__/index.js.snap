// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> should render the component 1`] = `
<h1
  styleName="container"
>
  React Starter, 
  Test app
</h1>
`;
